testsuite/layers/src/test/java/org/jboss/as/test/layers/LayersTestCase.java

```diff
@@ -32,17 +32,13 @@ public class LayersTestCase {
         "org.jboss.as.threads",
         "sun.jdk",
         "sun.scripting",
-        // No patching modules in layers
+        // TODO why doesn't core-tools provision these given that passive+ is used?
```

Comment:
@bstansberry , org.jboss.as.cli JBoss Modules module has an optional dependency on patching. Optional dependencies in module.xml are ignored to identify the Galleon package dependencies tree. No layer brings patching, so patching is not provisioned.

Reply:
Thanks @jfdenise. We talked about that but I forgot to come back and remove the comment.

Reply:
@jfdenise Fixed